app/models/invenio_rdm_record_converter.rb

```diff
@@ -631,8 +631,8 @@ def sizes
     end
   end
 
-  def user_email(user)
-    user.email == "joshelder@northwestern.edu" ? "JoshElder@northwestern.edu" : user.email
+  def user_email(email)
```

Comment:
might as well rename `user_email` to `normalize_email` or some such since `user` isn't related anymore.

Reply:
Good point

Reply:
agreed